samples/matter/lock/sample.yaml

```diff
@@ -53,14 +54,3 @@ tests:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp
-  sample.matter.lock.factory_data:
```

Comment:
You need to remove the entries in the quarantine configuration file here https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/quarantine_integration.yaml#L6 and here https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/quarantine_integration.yaml#L22